components/service-operator/apis/database/v1beta1/postgres_types.go

```diff
@@ -164,6 +167,7 @@ func (p *Postgres) GetStackTemplate() (*cloudformation.Template, error) {
 		},
 		DBSubnetGroupName:     cloudformation.Ref(DBSubnetGroupNameParameterName),
 		BackupRetentionPeriod: DefaultBackupRetentionPeriod,
+		EngineVersion:         p.Spec.AWS.EngineVersion,
```

Comment:
why do we have to specify engineversion at the cluster level when we didn't before?

Reply:
because the default value "latest" is no longer what we require for new resources ... we need to enable users to specify a version AND maintain compatibility with any old resources that did not specify an EngineVersion.